pd-client/rpc_worker.go

```diff
@@ -341,3 +345,45 @@ func (w *rpcWorker) checkResponse(resp *pdpb.Response) error {
 	}
 	return nil
 }
+
+func parseUrls(s string) ([]url.URL, error) {
+	items := strings.Split(s, ",")
+	urls := make([]url.URL, 0, len(items))
+	for _, item := range items {
+		u, err := url.Parse(item)
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+
+		urls = append(urls, *u)
+	}
+
+	return urls, nil
+}
+
+func rpcConnect(addr string) (net.Conn, error) {
+	req, err := http.NewRequest("GET", pdRPCPrefix, nil)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	urls, err := parseUrls(addr)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	for _, url := range urls {
+		conn, err := net.DialTimeout("tcp", url.Host, connectPDTimeout)
```

Comment:
If one url can not be connected, should we try another one instead of return error?